examples/feature_selection/plot_feature_selection.py

```diff
@@ -54,7 +54,8 @@
 scores = -np.log10(selector.pvalues_)
 scores /= scores.max()
 plt.bar(X_indices - .45, scores, width=.2,
-        label=r'Univariate score ($-Log(p_{value})$)', color='darkorange')
+        label=r'Univariate score ($-Log(p_{value})$)', color='darkorange',
```

Comment:
Again, I don't think you need to match the look everywhere. The point was to fix things that don't convey the intended meaning any more or look really bad. Sorry if I wasn't clear in the issue.